dotnet/src/Microsoft.AutoGen/RuntimeGateway.Grpc/Services/Grpc/GrpcWorkerConnection.cs

```diff
@@ -6,6 +6,15 @@
 
 namespace Microsoft.AutoGen.RuntimeGateway.Grpc;
 
+public static class ServerCallContextExtensions
```

Comment:
thanks for adding this - probably several other places we could compress/delete code and point to this.